tests/other/metadce/hello_libcxx_fastcomp_O2_EMULATED_FUNCTION_POINTERS.sent

```diff
@@ -22,6 +22,7 @@ _fd_write
 _getenv
 _llvm_stackrestore
 _llvm_stacksave
+_llvm_trap
```

Comment:
Wait: wasn't this supposed to become a wasm `(unreachable)` opcode? This suggests this is generating a function?

Reply:
That's just on fastcomp, which doesn't map it optimally like the LLVM wasm backend does.